lib/sentry/client_error.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Sentry.ClientError do
+  @moduledoc """
+  An exception struct that represents an error returned by Sentry.
```

Comment:
What do you think about

```suggestion
  An exception struct that represents an error returned by Sentry when
  reporting an error or a message.
```

?

Reply:
Yep yep!